practices/libraries.md

```diff
@@ -0,0 +1,99 @@
+---
+slug: libraries
+stages:
+  - development
+short_description: Updating dependencies from your libraries can be a huge pain. But it is a necessary pain! Your whole project can be jeopardized without regular updates. Automatize and keep track of the dependencies’ changes.
+tags:
+  - libraries
+  - lockfile
+  - dependencies
+  - improvement
+  - functionality
+  - automated deployment
+  - continuous integration
+  - continuous delivery
+  - continuous deployment
+  - unit tests
+keywords:
+  - software library
+  - updating software
+  - updating dependencies
+  - updating libraries
+covered_by_dx_scanner: true
+---
+
+# Libraries
+
+**TL;DR**
+
+Updating dependencies from your libraries can be a huge pain. But it is a necessary pain! Your whole project can be jeopardized without regular updates. Automatize and keep track of the dependencies’ changes.
+
+## What Are the Libraries
+
+Libraries collect resources for software development. They consist of pre-written code, procedures, classes, scripts, data configurations, and more. Developers add libraries to the program they are developing to get some functionalities, dependencies, or automation without having to program it specifically for their program.
```

Comment:
> Libraries collect resources for software development.

I am not sure if libraries "collect" resources. It's more a part of code to be reused in the developer's application.

Reply:
What about "Libraries cover resources" or "Libraries cover software development requirements"?

Reply:
`Libraries consist...`